src/mds/MDSRank.cc

```diff
@@ -2497,6 +2499,42 @@ bool MDSRank::command_dirfrag_ls(
   return true;
 }
 
+void MDSRank::command_openfiles_ls(Formatter *f) 
+{
+  Mutex::Locker l(mds_lock);
+  
+  for (auto p : mdlog->get_segments()) {
+    LogSegment *ls = p.second;
+    auto q = ls->open_files.begin(member_offset(CInode, item_open_file));
+    while (!q.end()) {
+      CInode *in = *q;
+      ++q;
+      if (in->last == CEPH_NOSNAP) {
```

Comment:
I think we will still want to show snapshotted inodes that are open?

Reply:
that's ok